pkg/api/handlers/libpod/containers.go

```diff
@@ -63,11 +62,14 @@ func ListContainers(w http.ResponseWriter, r *http.Request) {
 	decoder := r.Context().Value("decoder").(*schema.Decoder)
 	query := struct {
 		All       bool                `schema:"all"`
+		External  bool                `schema:"external"`
```

Comment:
I think this requires swagger changes as well to document the new parameters. I couldn't find `external` mentioned.

Reply:
Added.